### DISCUSSION TITLE
1 source to multiple destinations

### DISCUSSION THREAD

**Discussion by ro8inmorgan** [airbytehq/airbyte#37738] [Connector Ideas and Features]
A nice feature would be able to configure multiple destinations in 1 connector. The reasoning behind this is that some of our sources would not be happy if we connect multiple times parallel to extract the databases but we do need this data in multiple destinations. I think technically it would be possible for a source connector to send it's data to multiple destination connectors?

**Comment by akozichev** [comment]
you might want to use some staging destination for this like Amazon S3 or MinIO. Then you can use it as a source for multiple destinations.

[a comment by an automated account is not shown]
